TFLite/src/Graph.py

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Are the Edge, Node, Graph classes specific to tflite or apply to other input types like graphdef also? In that case, it might be better to put it in a parent directory or subdirectory like 'common' or something similiar

Reply:
Added a common sub-directory in src for the same